package.json

```diff
@@ -10,13 +10,14 @@
   },
   "scripts": {
     "build": "yarn tsc",
-    "test": "snyk test"
+    "test": "snyk test",
+    "test:jest": "backstage-cli test"
```

Comment:
To run jest tests in the project we use the backstage-cli.

Reply:
Have you considered replacing `npm test` with this new command? I haven't noticed `npm test` being called anywhere, so it seems like this addition was primarily for the convenience of quickly invoking `snyk test`. `npm test` is typically the standard for running tests, so I suggest we consider changing this behavior.